packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -44,52 +28,52 @@ export type QueryContextType<
   removeQueryFilter: ImmutableTableQueryResult['removeQueryFilter']
   /** Removes a value from a QueryFilter. If no more values remain in the filter, the filter is also removed */
   removeValueFromQueryFilter: ImmutableTableQueryResult['removeValueFromQueryFilter']
-  /** The error returned by the query request, if one is encountered */
-  error: SynapseClientError | null
-  /** The status of the asynchronous job. */
-  asyncJobStatus?: AsynchronousJobStatus<QueryBundleRequest, QueryResultBundle>
   /** Whether facets are available to be filtered upon based on the current data */
-  isFacetsAvailable: boolean
+  hasFacetedSelectColumn: boolean
   /** Returns true iff the current request has resettable filters applied via facet filters or additionalFilters. Excludes filters applied to a locked column */
   hasResettableFilters: boolean
-  getColumnModel: (columnName: string) => ColumnModel | null
```

Comment:
Was only used in one place, and does not make sense to have here now that query data isn't actually stored here anymore.